04.ls/ls.rb

```diff
@@ -1,23 +1,29 @@
 # frozen_string_literal: true
 
+require 'optparse'
 COLUMN_COUNT = 3
 SPACE = 1
 
 def main
+  options = {}
+  opt = OptionParser.new
+  opt.on('-a') { |v| v }
+  opt.parse!(ARGV, into: options)
+
   path =  "#{Dir.getwd}/"
   entries = Dir.entries(path).sort
-  visible_entries = entries.delete_if { |entry| entry.start_with?('.') }
-  marked_entries = add_mark_end_of_entries(visible_entries, path)
-  row_count = (marked_entries.length.to_f / COLUMN_COUNT).ceil
-  aligned_entries = align_entries(row_count, marked_entries)
+  filtered_entries = options.key?(:a) ? entries : entries.reject { |entry| entry.start_with?('.') }
+  entries_have_suffix = append_suffix_by_file_type(filtered_entries, path)
```

Comment:
```suggestion
  entries_with_suffix = append_suffix_by_file_type(filtered_entries, path)
```

こっちの方が英語的には自然かもしれません